CONTRIBUTING.md

```diff
@@ -1,53 +1,59 @@
 Contributing to Rust-Lightning
 ==============================
 
-The Rust-Lightning project operates an open contributor model where anyone is
-welcome to contribute towards development in the form of peer review, documentation,
-testing and patches.
+The `rust-lightning` project operates an open contributor model where anyone is
+welcome to contribute towards development in the form of peer review,
+documentation, testing and patches.
 
-Anyone is invited to contribute without regard to technical experience, "expertise", OSS
-experience, age, or other concern. However, the development of cryptocurrencies demands a
-high-level of rigor, adversarial thinking, thorough testing and risk-minimization.
-Any bug may cost users real money. That being said, we deeply welcome people contributing
-for the first time to an open source project or pick up Rust while contributing. Don't be shy,
-you'll learn.
+Anyone is invited to contribute without regard to technical experience,
+"expertise", OSS experience, age, or other concern. However, the development of
+cryptocurrencies demands a high-level of rigor, adversarial thinking, thorough
+testing and risk-minimization. Any bug may cost users real money. That being
```

Comment:
"risk-minimization engineering process"

Reply:
Hm, I don't think adding this checks out? Would read "However, the development of cryptocurrencies demands a high-level of (...) risk-minimization engineering process". So I think the current form is more correct?

Reply:
Sure, just risk-minimization doesn't seem to refer what is actually qualified risk-minimized.